src/OpenApi/gen/XmlComments/MemberKey.cs

```diff
@@ -46,7 +46,7 @@ public static MemberKey FromMethodSymbol(IMethodSymbol method, Compilation compi
 
             returnType = actualReturnType.TypeKind == TypeKind.TypeParameter
                 ? "typeof(object)"
-                : $"typeof({actualReturnType.ToDisplayString(_typeKeyFormat)})";
+                : $"typeof({ReplaceGenericArguments(actualReturnType.ToDisplayString(_typeKeyFormat))})";
```

Comment:
We need to make sure that all generics are emitted as open generics for method declarations. We already do this elsewhere in the `MemberKey` for the declaring type and the property type but not for methods.